### DISCUSSION TITLE
Exciting News! School or College Management System is Now Part of GSSoC'24!

### DISCUSSION THREAD

**Discussion by harxh-v** [harxh-v/StreamlineEDU#38] [Announcements]
We're thrilled to announce that project has been selected as a participating project for Girl Script Summer of Code 2024 (GSSoC'24)!

This is a fantastic opportunity for aspiring developers to contribute to open source and gain valuable experience.

**Comment by CarlosSuperAdmin** [comment]
Congratulations !! I´m loving your plugin.

**Reply by harxh-v** [reply]
@CarlosSuperAdmin 

> Congratulations !! I´m loving your plugin.

Thank you for your feedback. I'd love to hear from you when you encounter an issue, or find a need for a new feature.

Feel free to reach me out.
__
Thanks!
